e2e/_suites/fleet/features/system_integration.feature

```diff
@@ -0,0 +1,224 @@
+@deploy-system_integration
+Feature: System Integration package metrics
+Scenarios for System Integration logs and metrics packages.
+
+@deploy-system_integration-with-core
+Scenario Outline: Adding core System Integration to an Policy
+  Given a "<os>" agent is deployed to Fleet with "tar" installer
+    And the agent is listed in Fleet as "online"
+  When the policy is updated to have "system/metrics" set to "core"
+    And we verify that "system/metrics" with "core" metrics in the datastreams
```

Comment:
I'd rename the step to remove the `we` part:
```suggestion
    And "system/metrics" with "core" metrics are present in the datastreams
```
Note that you should update all scenarios and the Go code that glues the step name with the method. Perfect if you also rename the method too

Reply:
Done